aten/src/ATen/cpu/vec256/vec256_qint.h

```diff
@@ -209,7 +205,10 @@ inline void __attribute__((always_inline)) QuantizeAvx2(
     // Note that we cannot implement the same behavior as the vectorized code
     // using std::round because it does rounding away from zero in halfway
     // cases.
-    dst[i] = nearbyint(clipped);
+    transformed = zero_point + nearbyint(transformed);
```

Comment:
Based on the comment above, why cant we force the rounding mode to be FE_TONEAREST?

Reply:
> This is the default mode. It should be used unless there is a specific need for one of the others.
> You should avoid changing the rounding mode if possible. It can be an expensive operation; also, some hardware requires you to compile your program differently for it to work. The resulting code may run slower. 
https://www.gnu.org/software/libc/manual/html_node/Rounding.html

Reply:
Can we atleast check what the rounding mode is print a warning?

Reply:
Is there a central init routine where we can add such a warning? cc: @jerryzh168

Reply:
cc @dzhulgakov